app/helpers/mailing_lists_helper.rb

```diff
@@ -31,7 +31,7 @@ def format_mailing_list_preferred_labels(mailing_list)
 
   def button_toggle_subscription
     if entry.subscribed?(current_user)
-      button_unsubscribe
+      button_unsubscribe if entry.subscriptions.where(subscriber_id: current_user.id, subscriber_type: Person.sti_name).exists?
```

Comment:
Dasselbe wird doch schon in `button_unsubscribe` geprüft und sollte ich nicht nochmals nötig sein?